src/server/app.js

```diff
@@ -347,29 +347,122 @@ app.post('/reject', async (req, res) => {
 
 // Generate Results page
 app.get('/location', async (req, res, next) => {
+  //get latitude and longitude from the parameter info
   const { lat, lng } = req.query
   console.log('lat = ', lat)
   console.log('lng = ', lng)
   if (lat === undefined || lng === undefined) {
     return res.redirect('/404')
   }
+  //Get GIS identifiers for location
   const gisResponse = await gisLocationQuery(lat, lng).catch((err) => {
     console.error(err)
     return res.status(500).send('500')
   })
-  console.log(gisResponse)
+  console.log("GIS Response: " + gisResponse)
+  
+  
+  
+  //Get info for GIT identifiers
+  const queryString = 'SELECT a.gisidentifier, b.officetitle, c.name, c.holderid, d.levelnum FROM Locations a, Offices b, OfficeHolders c, LocationTypes d WHERE a.gisidentifier = ANY ($1) AND a.locationid = b.locationid AND b.currentholder = c.holderid AND a.typeid = d.typeid';
+  
+  var response = String(gisResponse);
+  let gisIdentifiers = response.split(',');
+  
+  const locationRes = await databasePool.query(queryString, [gisIdentifiers]).catch((err) => {
+    console.error(err)
+    return res.status(500).send('A server error occurred, please try again')
+  })
```

Comment:
can you merge master into your branch and move your querying logic into the sql.js file? We don't want to clutter the router file. Also in `sql.js` we can reuse the same database pool.